tools/vms-generator/utils/utils.go

```diff
@@ -833,7 +833,7 @@ func GetVMIWithHookSidecar() *v1.VirtualMachineInstance {
 	addNoCloudDiskWitUserData(&vmi.Spec, "#cloud-config\npassword: fedora\nchpasswd: { expire: False }")
 
 	vmi.ObjectMeta.Annotations = map[string]string{
-		"hooks.kubevirt.io/hookSidecars":              `[{"image": "registry:5000/kubevirt/example-hook-sidecar:devel"}]`,
+		"hooks.kubevirt.io/hookSidecars":              `[{"image": "kubevirt/example-hook-sidecar:devel"}]`,
```

Comment:
I think we're generating this annotation wrong. The example should be using the DockerPrefix value like all the other containers in our generation code. Essentially, this shouldn't have been hardcoded to registry:5000 to begin with.